### PR TITLE
Cart and wishlist

### PR DESCRIPTION
**Pull request by rdthetha**
added cart and wishlist

### CODE REVIEW

File: cart.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
+.ec-cart-container{
+
```
Comment (varun1233) [inline cart.css]: Remove unwanted space

File: cart.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+}
+.card-container-horizontal{
+
+    min-width: 32rem;
+    max-width: 35rem;
+    min-height: 18rem;
+    justify-content: flex-start;
+    background-color: var(--ec-secondary-background);
+}
+.img-contain{
```
Comment (varun1233) [inline cart.css:24]: Can't these be merged together?
.img-contain and .image-contain img

File: cart.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+/*COUNTER CSS*/
+
+.ec-product-quantity{
+
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin-top: 0.5rem;
```
Comment (varun1233) [inline cart.css:43]: use px for values less than 1 rem

File: cart.html [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+                </ul>
+            </nav>
+        </header>
+        <div class="h3 center-text ec-head-text">My Cart!</div>
+        <div class="ec-cart-container">
+            
+            <div class="ec-product-card-cart">
+                <div class="card-container-horizontal">
+                    <div class="img-contain">
+                        <img src="image/run-shoes.jpg" alt="Photo">
```
Comment (varun1233) [inline cart.html]: Trying using relevant naming for in attribute

File: cart.css (hunk 1 above)
Comment (rdthetha) [reply cart.css]: would remove it!

File: cart.html (hunk 4 above)
Comment (rdthetha) [reply cart.html]: would fix the alt attribute!

File: cart.css (hunk 3 above)
Comment (rdthetha) [reply cart.css:43]: thank u for the suggestion!

File: wishlist.css [hunk 5]
```diff
@@ -0,0 +1,27 @@
+.card-container{
+
+    min-width: 22rem;
+    height: 27rem;
+}
+
+.img-contain>img{
+
```
Comment (varun-Tad) [inline wishlist.css:8]: Remove unwanted space. Otherwise the code looks good to merge

Comment (rdthetha) [reply wishlist.css:8]: sure!

**Pull request merged**